src/pages/Accounts/Modals/SendFunds.tsx

```diff
@@ -147,7 +151,6 @@ export const AskTransferFundsModal: FC<AskSendFundsModalProps> = ({ isVisible, o
 
   const onChangeAddress = useCallback((input) => {
     setRecipientAddress(input);
-    getKusamaFee()();
```

Comment:
keep it here

you an check for input value there instead of using useEffect

Reply:
changed